scripts/basic-workflow-tests.sh

```diff
@@ -208,11 +219,15 @@ test_exec() {
     task Stopping the ilab model serve
     step Kill ilab model serve $PID
     kill $PID
+
+    task Evaluating a model
+    test_evaluate
```

Comment:
Didn't this need to be moved up to take effect?

Reply:
thanks for remembering this comment from eariler, but it's good now as of https://github.com/instructlab/instructlab/pull/1580

It used to always exit before here. Now it will make it this far as long as `-f` (full training) is used. That does mean that `-e` without `-f` will not work. It's probably worth a check for that. I've been adding similar checks to `set_defaults`.